commands/types/map/mapget.go

```diff
@@ -57,7 +57,7 @@ var mapGetCmd = &cobra.Command{
 		if value != nil {
 			switch v := value.(type) {
 			case serialization.JSON:
-				if err := quick.Highlight(os.Stdout, v.String(),
+				if err := quick.Highlight(os.Stdout, fmt.Sprintln(v.String()),
```

Comment:
I don't understand why you wrap the string around the `Sprintln`. The new [PR](https://github.com/utku-caglayan/hazelcast-commandline-client/blob/improvements-1/commands/types/map/mapget.go#L59) is handling other way around.